packages/client/src/components/ToggleEntryEnabled.component.tsx

```diff
@@ -0,0 +1,64 @@
+import { Switch } from '@mui/material';
+import { useEffect } from 'react';
+import { useIsEntryEnabledLazyQuery, useSetEntryEnabledMutation } from '../graphql/tag/tag';
+import { useStudy } from '../context/Study.context';
+import { useConfirmation } from '../context/Confirmation.context';
+
+export default function ToggleEntryEnabled(props: { entryId: string }) {
+  const [isEntryEnabled, isEntryEnabledResults] = useIsEntryEnabledLazyQuery();
+  const [setEntryEnabledMutation, setEntryEnabledResults] = useSetEntryEnabledMutation();
+
+  const { study } = useStudy();
+  const confirmation = useConfirmation();
+
+  useEffect(() => {
+    if (study) {
+      isEntryEnabled({
+        variables: {
+          study: study._id,
+          entry: props.entryId
+        },
+        fetchPolicy: 'network-only'
+      });
+    }
+  }, [study, setEntryEnabledResults.data]);
+
+  useEffect(() => {
+    if (setEntryEnabledResults.called) {
+      if (setEntryEnabledResults.error) {
+        //show error message
+        console.log('error toggling entry', setEntryEnabledResults.error);
```

Comment:
NIT: Best to use a `console.error` and the update list should be for `setEntryEnabledResults.error` instead since we would wanna capture if an error takes lace